src/arch/armv8/gic.c

```diff
@@ -68,8 +68,10 @@ void gicd_init()
 
     /* No need to setup gicd->NSACR as all interrupts are  setup to group 1 */
 
-    interrupts_reserve(platform.arch.gic.maintenance_id,
-                       gic_maintenance_handler);
+    if(!interrupts_reserve(platform.arch.gic.maintenance_id,
+                       gic_maintenance_handler)) {
+        ERROR("Failed to assign GIC maintenance interrupt");
```

Comment:
```suggestion
        ERROR("Failed to reserve GIC maintenance interrupt");
```

Reply:
Thank you, included in https://github.com/bao-project/bao-hypervisor/pull/97/commits/878ad2b2ed85b8ed383a63e32650ee9363c624d8.